### ISSUE TITLE
Missing prop 'alignDotsOnXAxis' on documentation

### ISSUE THREAD

**Issue by sergiosilvajr** [tsepeti/react-native-dots-pagination#8]
I have seen this prop and its use is really helpful. It saved me a lot of time!

Could you add it on the docs?

**Comment by yasaricli** [comment]
thank you for feedback @sergiosilvajr 

You can specify what you use the `alignDotsOnXAxis` parameter for and add it.

We are waiting for a pull request

Thank you.

**Comment by timothyerwin** [comment]
yeah, this should be default true...I was about to trash the component because it was looking weird
